src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,186 @@
+﻿project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-10-18 #}
+{# wf_published_on: 2017-10-18 #}
+
+# Graphical Content {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+Web users are a visual bunch, and we rely on images to support web content. Like text, graphical content is a critical component in conveying information in a web page or app. All kinds of images, from charts and graphs to icons and arrows to mugshots and maps, provide instantaneous data and improve reader comprehension and retention. But unlike text, images require considerable time and bandwidth to download and render. Graphical content can easily account for 60%-85% of a typical website's total bandwidth (source: it depends on who you ask).
```

Comment:
It might be worth referencing a statistic from one of our team's maintained resources here :) HTTPArchive notes that the average site includes approximately 50%+ of images: http://httparchive.org/interesting.php. That's more than any other type of resource that consumes user bandwidth.